src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -234,12 +239,12 @@ public:
     } // End of onExecute() method
 };
 
-
 template<class ImplementationType>
-inline ISimulation<ImplementationType>::ISimulation(Data::Study& study,
-    const ::Settings& settings,
-    Benchmarking::IDurationCollector& duration_collector,
-    IResultWriter& resultWriter) :
+inline ISimulation<ImplementationType>::ISimulation(
```

Comment:
Here ISimulation constructor : 
We would like the first arg right after the opening "(", but in this case, all args cannot start at the same column : the **duration_collector** arg is too long and exceeds 100 columns.
As a consequence, the first arg is rejected at the next line, so that all args can start at the same column.
But all args start after 2 spaces. We could prefer more indentation, like 4 spaces. But the args would start at the same column as the initialization list. 
So why not 8 spaces ?

Reply:
I don't like this formatting too much either. However there is only ONE parameter `ContinuationIndent` which decides for all types of continuation indent.
We need to check it's acceptable to use 8 for everything.